### PR TITLE
feat-606: Add manual approval in e2e test ci

### PR DESCRIPTION
**Pull request by nightfury1204**
https://github.com/convox/issues-private/issues/606

ci : https://github.com/convox/convox/actions/runs/3045546296

https://user-images.githubusercontent.com/12232198/189977609-ce6d6bd6-3d1c-49cb-aa5d-4a519fdbf2cc.mp4

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -155,24 +145,94 @@ jobs:
...
-      - name: before version upgrade/downgrade setup and test
+      - name: test
+        run: ci/test.sh
+      - name: debug
+        run: ci/debug.sh
+        if: always()
+      - name: uninstall
+        run: ci/uninstall.sh
+        if: always()
+  ci-downgrage:
```
Comment (Twsouza) [inline .github/workflows/release.yml:158]:
```suggestion
  ci-downgrade:
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -155,24 +145,94 @@ jobs:
...
+    if: github.repository == 'convox/convox'
+    environment: 'UpgradeDowngradeTests'
```
Comment (Twsouza) [inline .github/workflows/release.yml:160]: Can `UpgradeDowngradeTests` be renamed? In case we want to add more additional tests...

Comment (nightfury1204) [reply .github/workflows/release.yml:160]: this environment is configured by @heronrs, i don't have access. He can give more context on it

File: .github/workflows/release.yml (hunk 1 above)
Comment (nightfury1204) [reply .github/workflows/release.yml]: nice catch. fixed it

**Review by Twsouza — APPROVED** [review]
👍

File: .github/workflows/release.yml (hunk 2 above)
Comment (heronrs) [reply .github/workflows/release.yml:160]: Yeah sorry, that was a bad name. I can't rename the environment sadly but I'll create a new one later. Maybe name it just ManuApprovalTests.

Merging for now just so we can move forward

**Pull request merged**